sdks/golang/docs/PipelineSourceOneOf2.md

```
@@ -0,0 +1,93 @@
+# PipelineSourceOneOf2
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**AcceptedFormat** | **string** |  | 
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)